Marlin/src/lcd/menu/menu_advanced.cpp

```diff
@@ -511,6 +511,9 @@ void menu_backlash();
     #else
       const xyze_ulong_t &max_accel_edit_scaled = max_accel_edit;
     #endif
+    #if ENABLED(DEFAULT_ACCELERATION_SPINDLE)
+      constexpr uint32_t accel_edit_spindle = 99000;
```

Comment:
The type needs to be a uint16_t just as `SpindleLaser::acceleration_spindle_deg_per_s2. The value of 99000 is out of the uint16_t range.